server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -59,33 +59,42 @@ export class TenantManager implements core.ITenantManager {
 		documentId: string,
 		includeDisabledTenant = false,
 	): Promise<core.ITenant> {
-		const restWrapper = new BasicRestWrapper();
 		const [details, gitManager] = await Promise.all([
-			restWrapper.get<core.ITenantConfig>(`${this.endpoint}/api/tenants/${tenantId}`, {
-				includeDisabledTenant,
-			}),
-			this.getTenantGitManager(tenantId, documentId, includeDisabledTenant),
+			this.getTenantConfig(tenantId, includeDisabledTenant),
+			this.getContainerGitManager(tenantId, documentId, undefined, includeDisabledTenant),
 		]);
 
 		const tenant = new Tenant(details, gitManager);
 
 		return tenant;
 	}
 
-	public async getTenantGitManager(
+	public async getTenantConfig(
+		tenantId: string,
+		includeDisabledTenant = false,
+	): Promise<core.ITenantConfig> {
+		const restWrapper = new BasicRestWrapper();
+		return restWrapper.get<core.ITenantConfig>(`${this.endpoint}/api/tenants/${tenantId}`, {
+			includeDisabledTenant,
+		});
+	}
+
+	public async getContainerGitManager(
```

Comment:
Assuming "Container" here is in the context of Azure Storage Container, replacing the "Tenant" with "Container" does not seem to be a good decision for 2 reasons:

1. From the gitManager consumers' perspective it is still a tenant scoped storage proxy and the fact that in FRS those storage requests will be differentiated based on documentId is just a FRS specific implementation detail which does not need to be surfaced here.
2. In this context, "Container" is an FRS implementation specific name used for what is generally called as "Document" in OSS/FFX. Also, Container has a different meaning in FFX and using it here might be confusing.

Why not name it gitDocumentGitManager for consistency ? Or just a getGitManager maybe?

Reply:
I can do:
1. unchange the name to avoid break changes
2. Make change to use `gitDocumentGitManager` so it more close to the storage per document concept. The reason I choose the word `contaienr` instead of `document` is I remember we expose the concept of `container` to out side world? Just to double confirm, I can go with either.

Reply:
Either options is fine, just not Container :) The word Container is already used with 3 different but related meanings:

1. FFX [container](https://fluidframework.com/docs/glossary/#container)
2. The resource type in control plane - FluidRelayContainer
3. The Azure Storage Container

Reply:
Changed to `getDocumentGitManager`, I am eventually undo the change....